api/agent/agent_test.go

```diff
@@ -265,6 +270,10 @@ func TestCallConfigurationModel(t *testing.T) {
 		expectedHeaders.Add(k, v)
 	}
 
+	// These should be here based on payload length and/or fn_header_* original headers
+	expectedHeaders.Add("Content-Type", contentType)
```

Comment:
`expectedHeaders.Set` is more appropriate, `.Add` will turn header value into a set